manifests/config.pp

```diff
@@ -1,5 +1,8 @@
 # Configure dns
 class dns::config {
+  validate_array($dns::forwarders)
```

Comment:
I opted to put them in config.pp as IMHO, it looked cleaner.
However, I have no problem moving them, your call.